web-registry/src/components/templates/PortfolioTemplate.tsx

```diff
@@ -46,8 +46,12 @@ export const PortfolioTemplate: React.FC<PortfolioTemplateProps> = ({
   useEffect(() => {
     if (!ledgerRestUri || !accountAddress) return;
     const fetchData = async (): Promise<void> => {
-      const credits = await getEcocreditsForAccount(accountAddress);
-      setCredits(credits);
+      try {
```

Comment:
when working on this story I was getting some weird behavior from my REST endpoints, so I added some basic error handling + try/catch blocks